.github/workflows/release.yaml

```
@@ -6,55 +6,25 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: "3.10"
-      - run: pip install -e .[dev]
-      - run: pytest
+      - uses: actions/checkout@v5
+      - name: Setup uv
+        uses: astral-sh/setup-uv@v1
```

Please use `v7` for all actions
```suggestion
        uses: astral-sh/setup-uv@v7
```